CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
 #Package information
 hyro_init_package(signal_generator VERSION 0.0.1)
 
-find_package ( Qt4 COMPONENTS QtCore QtGui QtOpenGl )
+#find_package ( Qt4 COMPONENTS QtCore QtGui QtOpenGl )
```

Comment:
Remove those lines, not just comment them

Reply:
I fixed this ;)